src/content/docs/setup/configuration/multistore-setup.mdx

```diff
@@ -0,0 +1,291 @@
+---
+title: Multi-Store Implementation Walkthrough
+description: A hands-on guide demonstrating the complete multi-store setup process from file structure creation to store switcher configuration.
+---
+
+import { Steps } from '@astrojs/starlight/components';
+import Tasks from '@components/Tasks.astro';
+import Task from '@components/Task.astro';
+import { Code } from '@astrojs/starlight/components';
+import Aside from '@components/Aside.astro';
+
+This tutorial provides a comprehensive walkthrough of setting up multi-store functionality for an Adobe Commerce storefront using Edge Delivery Services. It demonstrates the complete process from initial planning to final validation, using a real-world scenario where the Acme brand expands from the United States to the Canadian market.
+
+## Overview
+
+In this walkthrough, we'll demonstrate how to:
+- Set up the necessary file structures for multi-store views
+- Configure store views in Adobe Commerce Admin
+- Update storefront configuration files
+- Preview and publish localized content
+- Configure the store switcher for proper store view display
+
+## Scenario
+
+The Acme brand is expanding its international presence and wants to target the Canadian market with two new locales:
+- **Canadian English** (en_ca)
+- **Canadian French** (fr_ca)
+
+## Prerequisites
+
+Before beginning this implementation, ensure:
+- Your storefront repository is linked to Document Authoring environment (da.live) via Edge Delivery Services
+- You have access to Adobe Commerce Admin for store view configuration
+- Your `fstab.yaml` file is properly configured
+- Your `config.json` file is linked to the Adobe Commerce instance
+
+## Implementation Steps
+
+<Tasks>
+
+<Task>
+### Verify Environment Setup
+
+Before implementing multi-store functionality, confirm your environment is properly configured.
+
+<Steps>
+1. **Check the `fstab.yaml` file** in your repository to ensure it's linked to the Document Authoring environment at da.live.
+
+2. **Verify the `config.json` file** is connected to your Adobe Commerce instance to enable content preview, publishing, and editing.
+</Steps>
+
+<Aside type="caution" title="Environment setup">
+This verification step is crucial as it ensures you can properly preview and publish content for your new store views.
+</Aside>
+
+</Task>
+
+<Task>
+### Create Content Folder Structure
+
+Set up the folder structure in your Document Authoring environment to support the new store views.
+
+<Steps>
+1. **Navigate to the connected content folder** in da.live.
+
+2. **Create the Canadian English folder**:
+   - Select "New folder"
+   - Name it `en_ca`
+
+3. **Create the French folder**:
+   - Select "New folder"  
+   - Name it `fr_ca`
+
+4. **Copy content from the main root folder** into each new store view folder:
+   - Select all content from the main root folder
+   - Copy the content
+   - Navigate to the `en_ca` folder and paste
+   - Repeat for the `fr_ca` folder
+</Steps>
```

Comment:
Is this true? Aren't we recommending using the Translate App instead of these manual steps?

Reply:
Removed